azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,88 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps[
+        "iot hub state"
+    ] = """
+        type: group
+        short-summary: Manage the state of an IoT Hub.
+    """
+
+    helps[
+        "iot hub state export"
+    ] = """
+        type: command
+        short-summary: Export the state of an IoT Hub to a file.
+        long-summary: The exported state will include hub configurations, edge deployments, device identities and twins, and module
```

Comment:
here configurations and edge deployments are separately called out where in the aspects enum there is only configurations

Reply:
I will reword to have ()